### PR TITLE
Add an optional table format for genome attributes return

### PR DESCRIPTION
**Pull request by MrCreosote**
Default to table since the payload is ~1/2 the size

### CODE REVIEW

File: src/service/data_products/genome_attributes.py [hunk 1]
```diff
@@ -140,11 +158,24 @@ async def get_ranks(
...
+        if output_table:
+            data.append([d[k] for k in sorted(_remove_keys(d))])
+        else:
+            data.append({k: d[k] for k in sorted(_remove_keys(d))})
+    fields = []
+    if d:
+        if sort_on not in d: 
+            raise errors.IllegalParameterError(
+                f"No such field for collection {collection_id} load version {load_ver}: {sort_on}")
+        fields = [{"name": k} for k in sorted(d)]
```
Comment (Tianhao-Gu) [inline src/service/data_products/genome_attributes.py:177]: why not just assign `fields = sorted(d)`?

**Review by Tianhao-Gu — APPROVED** [review]
just have one question. Otherwise, looks good to me.

File: src/service/data_products/genome_attributes.py (hunk 1 above)
Comment (dauglyon) [reply src/service/data_products/genome_attributes.py:177]: I think this is to allow adding other field properties (i.e. sort/filter type, human-readable name) in the future without it being a breaking change (correct me if I'm wrong @MrCreosote)

Comment (MrCreosote) [reply src/service/data_products/genome_attributes.py:177]: right

Comment (Tianhao-Gu) [reply src/service/data_products/genome_attributes.py:177]: 👍

**Pull request merged**